persistence/soft-index/pom.xml

```diff
@@ -81,6 +81,58 @@
                </instructions>
             </configuration>
          </plugin>
+         <plugin>
+            <groupId>org.codehaus.mojo</groupId>
+            <artifactId>exec-maven-plugin</artifactId>
+            <configuration>
+               <workingDirectory>${project.build.directory}</workingDirectory>
+            </configuration>
+            <executions>
+               <execution>
+                  <id>defaults</id>
+                  <phase>process-classes</phase>
+                  <goals>
+                     <goal>java</goal>
+                  </goals>
+                  <configuration>
+                     <mainClass>org.infinispan.commons.configuration.defaults.DefaultAttributeExtractor</mainClass>
```

Comment:
Maybe we should create a Maven plugin for it? This parts is being copied all over the place.